src/Select2/Select2.tsx

```diff
@@ -111,20 +129,31 @@ const Select2: React.FC<Props> = ({
     },
   });
 
+  const isInError =
+    requirement === FormElementRequirement.REQUIRED &&
+    (hasBeenFocused ? !selectedItem : initialIsInError);
   const id = name;
   const inputRef = useRef<HTMLInputElement>();
   return (
     <div className={classNames(cssClass.CONTAINER, formElementSizeClassName(size), className)}>
-      <label
-        className={classNames(cssClass.LABEL, hideLabel && cssClass.LABEL_HIDDEN)}
-        {...getLabelProps()}
-      >
-        {label}
-      </label>
+      <div className={classNames(cssClass.INFO_ROW, hideLabel && cssClass.INFO_ROW_LABEL_HIDDEN)}>
+        <label
+          className={classNames(cssClass.LABEL, hideLabel && cssClass.LABEL_HIDDEN)}
+          {...getLabelProps()}
+        >
+          {label}
+        </label>
+        {requirement && (
+          <label className={cssClass.INFO_REQUIREMENT} aria-live="polite" {...getLabelProps()}>
```

Comment:
Woa interesting, I didn't know `aria-live="polite" ` was a thing.

Reply:
Copied this from the `TextInput` component. Honestly I don't know how it fulfills a11y requirements https://github.com/Clever/components/blob/b54fd68cafa664519ca199853beb8554dc812966/src/TextInput/TextInput.tsx#L234 🤷‍♀️